appservice/src/createAppService/AppServicePlanSkuStep.ts

```diff
@@ -16,7 +16,11 @@ type ExtendedSkuDescription = WebSiteManagementModels.SkuDescription & { label?:
 
 export class AppServicePlanSkuStep extends AzureWizardPromptStep<IAppServiceWizardContext> {
     public async prompt(wizardContext: IAppServiceWizardContext): Promise<void> {
-        let skus: ExtendedSkuDescription[] = wizardContext.advancedCreation ? this.getCommonSkus() : this.getRecommendedSkus();
+        let skus: ExtendedSkuDescription[] = this.getRecommendedSkus()
+        if (wizardContext.advancedCreation) {
+            skus = skus.concat(this.getCommonSkus()).filter(sku => !(!sku.label && sku.name && sku.name === 'F1'));
```

Comment:
No need to do this filtering stuff when you can just remove `F1` from `getCommonSkus`

Also the name `getCommonSkus` doesn't really make sense anymore. Maybe `getAdvancedSkus`